.clang-format

```diff
@@ -3,39 +3,41 @@
 BasedOnStyle: LLVM
 ---
 Language: Cpp
-IndentWidth: 4
-IndentExternBlock: NoIndent
-BreakBeforeBraces: Custom
-BinPackArguments: false
-BinPackParameters: false
+Standard: Cpp11
```

Comment:
Unrelated to this PR's changes but, is there a reason why this is still set to Cpp11 instead of Cpp17?

Reply:
Not that I know of. I'll make a separate PR for that.